runtime/bcverify/vrfyhelp.c

```diff
@@ -278,7 +278,7 @@ buildStackFromMethodSignature( J9BytecodeVerificationData *verifyData, UDATA **s
 			arity++;
 		}
 
-		if (args[i] == 'L') {
+		if (J9_IS_OBJECT_OR_VALUETYPE(args[i])) {
```

Comment:
How is the verifier recording the difference between a Q and an L?  The code here is building an internal format for the stackmap which the code below doesn't differentiate between a Q & L.  I think this is incorrect and something (new tag type?) will be needed to handle this case.